app/src/main/res/layout/aims_layout.xml

```diff
@@ -5,10 +5,10 @@
     android:id="@+id/card_view"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_margin="10dp"
+    android:layout_margin="8dp"
```

Comment:
You can create dimens file in res/values and save all dimens into the file, then use dimens from that file

Reply:
ok knayem vonca arvum kanem